components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/utils/APIAndAPIProductCommonUtil.java

```diff
@@ -639,4 +647,74 @@ public static void addClientCertificates(String pathToArchive, APIProvider apiPr
             throw new APIImportExportException(errorMessage, e);
         }
     }
+
+    /**
+     * Get the subscription level policy names of an API/API Product
+     *
+     * @param apiTypeWrapper API or API Product to be exported
+     * @throws APIImportExportException
+     */
+    public static Set<String> getAvailableTierNames(ApiTypeWrapper apiTypeWrapper) {
+        Set<String> tiers = new LinkedHashSet<String>();
+        Set<Tier> tiersFromApi;
+        if (!apiTypeWrapper.isAPIProduct()) {
+            tiersFromApi = apiTypeWrapper.getApi().getAvailableTiers();
+        } else {
+            tiersFromApi = apiTypeWrapper.getApiProduct().getAvailableTiers();
+        }
+        for (Tier tier : tiersFromApi) {
+            tiers.add(tier.getName());
+        }
+        return tiers;
+    }
+
+    /**
+     * Set the subscription level tiers of an API/API Product by validating with the tiers available in the environment
+     *
+     * @param apiJsonContent JSON content of API or API Product to be imported
+     * @param apiProvider    API Provider
+     * @throws APIImportExportException
```

Comment:
APIManagementException is thrown, replace the APIImportExportException with it in the java docs.

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/commit/eb9b7181c0f6cb5b9624602d91d28a7479bf551b